test/e2e/azure_test.go

```diff
@@ -155,4 +155,39 @@ var _ = Describe("Workload cluster creation", func() {
 			})
 		})
 	})
+
+	Context("Creating a ipv6 control-plane cluster", func() {
```

Comment:
@mboersma do you have any thoughts on how to integrate this test better? right now the tests run serially and adding this third cluster create makes the presubmit take too long (~1h30m) so I'm tempted to disable it by default until we can reorganize the tests and run them in parallel

Reply:
@CecileRobertMichon I think these tests actually run in parallel. If I add the `-stream` flag to `ginkgo` and run e2e locally, I see them being provisioned at the same time:

```console
[1] INFO: Creating log watcher for controller system/capz-controller-manager, pod capz-controller-manager-6678f5d558-nx5j9, container kube-rbac-proxy
[1] Workload cluster creation Creating a single control-plane cluster 
[1]   With 1 worker node
[1]   /Users/matt/Projects/src/sigs.k8s.io/cluster-api-provider-azure/test/e2e/azure_test.go:73
[1] STEP: Creating a namespace for hosting the "create-workload-cluster" test spec
[2] Workload cluster creation Creating a ipv6 control-plane cluster 
[2]   With ipv6 worker node
[2]   /Users/matt/Projects/src/sigs.k8s.io/cluster-api-provider-azure/test/e2e/azure_test.go:160
[3] Workload cluster creation Creating highly available control-plane cluster 
[3]   With 3 control-plane nodes and 2 worker nodes
[3]   /Users/matt/Projects/src/sigs.k8s.io/cluster-api-provider-azure/test/e2e/azure_test.go:96
[3] STEP: Creating a namespace for hosting the "create-workload-cluster" test spec
[2] STEP: Creating a namespace for hosting the "create-workload-cluster" test spec
...
% az group list -o table
Name              Location    Status
----------------  ----------  ---------
NetworkWatcherRG  westeurope  Succeeded
capz-e2e-1t44a9   eastus      Succeeded
capz-e2e-f3zrec   eastus      Succeeded
capz-e2e-rsj3wv   eastus      Succeeded
```

So that *should* mean that the slowest of these three `It` clauses is the overall test time. @jsturtevant is that what you've seen?

Reply:
I opened https://github.com/kubernetes-sigs/cluster-api-provider-azure/issues/855 to track some improvements.  Attempted some changes in https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/856 but they did not work out.

Reply:
@mboersma it doesn't look like they are running in parallel in CI

<img width="1655" alt="Screen Shot 2020-09-15 at 12 08 43 PM" src="https://user-images.githubusercontent.com/8650424/93253842-4d057580-f74c-11ea-8bb5-2f0fc37ecee8.png">

Reply:
I cooked up a bunch of fake `It` specs that just do `time.Sleep()` and confirmed that ginkgo runs them in parallel from azure_test.go using ./scripts/ci-e2e.go. So maybe some logging will dispel the confusion: see #936.

Reply:
See kubernetes/test-infra#19275 and #939 for what I think is the right fix (which will also work with this PR).